docs/sources/alerting/fundamentals/notifications/group-alert-notifications.md

```diff
@@ -55,7 +55,7 @@ For example, given the `Group by` option set to the `team` label:
 
 ### Group by alert rule or labels
 
-By default, notification policies in Grafana **group alerts by the alert rule**—they are grouped by the `alertname` and `grafana_folder` labels, since alert names are not unique across folders.
+By default, notification policies in Grafana **group alerts by the alert rule**. Specifically, they are grouped using the `alertname` and `grafana_folder` labels, as alert rule names are not unique across folders.
```

Comment:
```suggestion
By default, notification policies in Grafana group alerts by alert rule. They are grouped using the `alertname` and `grafana_folder` labels, as alert rule names are not unique across folders.
```

Reply:
i think the bold was confusing - i guess its in the ui - but think we don't need to use it here

Reply:
addressed by https://github.com/grafana/grafana/pull/88656/commits/08648de8da2990ef99c4325134694ae48cb3f7a3